libs/wingsdk/src/cloud/function.ts

```diff
@@ -110,7 +110,7 @@ export abstract class Function extends Resource implements IInflightHost {
 
     const assetName = ResourceNames.generateName(this, {
       // Avoid characters that may cause path issues
-      disallowedRegex: /[><:"/\\|?*]/g,
+      disallowedRegex: /[><:"/\\|?* ]/g,
```

Comment:
tip: We can disallow all whitespace characters using a character class
```suggestion
      disallowedRegex: /[><:"/\\|?*\s]/g,
```

Reply:
Great callout!